AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/Support/EquatableExtensions.swift

```diff
@@ -37,7 +37,7 @@ extension ReconcileAndLocalSaveOperation.Action: Equatable {
                 && model1.model.modelName == model2.model.modelName
         case (.reconciled(let disposition1), .reconciled(let disposition2)):
             return disposition1 == disposition2
-        case (.applied(let model1), .applied(let model2)):
+        case (.applied(let model1, _), .applied(let model2, _)):
```

Comment:
It doesn't make sense to ignore the boolean value of this enum case when testing for equality. When would you expect enum cases with different booleans to still report as equal?

Reply:
fixed